Source: @pullmoll
Upstream: no
Reason: sanitizer does not link and is missing for musl; enable PIC

--- cpp/Makefile	2019-07-16 01:16:50.000000000 +0200
+++ cpp/Makefile	2020-01-29 22:57:19.157930868 +0100
@@ -29,12 +29,7 @@
 # - CXXFLAGS: Any extra user-specified compiler flags (can be blank).
 
 # Mandatory compiler flags
-CXXFLAGS += -std=c++11
-# Diagnostics. Adding '-fsanitize=address' is helpful for most versions of Clang and newer versions of GCC.
-CXXFLAGS += -Wall -fsanitize=undefined
-# Optimization level
-CXXFLAGS += -O1
-
+CXXFLAGS += -std=c++11 -Wall -fPIC
 
 # ---- Controlling make ----
 
